### ISSUE TITLE
3.8.2 broke my 2018 i9 mbp

### ISSUE THREAD

**Issue by Sebastian-blu** [osxfuse/osxfuse#520]
hey I just migrated things over from my 2016 mbp and updated fuse and gave it permission to run cause it was blocked. and when I restarted the computer can't get past the loading screen. I can't boot into safe mode either so i guess I have to boot into recovery and redo the whole process. But I am confused on what might have happened here? it was fine and now not so fine. This is a bit above my understanding so any advice will be greatly appreciated thanks

**Comment by bfleischer** [comment]
FUSE is a completely passive system extension. That means unless you are starting an app or manually mount a file system that requires FUSE, the FUSE kernel extension will not even be loaded and cannot interfere with your system. In any case it will first be loaded after your system hast successfully booted. I don't think FUSE is to blame for your current situation.

You can try to booting your Mac in safe mode and remove FUSE. If you cannot get into safe mode try booting into recovery. From there you can open a Terminal window and remove FUSE manually.

**Comment by Sebastian-blu** [comment]
yep your correct its not fuse, it happens whenever I allow an unidentified dev in the security and privacy settings. It happened with fuse first then I reinstalled the os from recovery mode and when it was finished fuse was working fine with pcloud. Then I allowed the dev of sound flower and the same thing happened it wouldn't boot after it restarted, and I had to reinstall macOS again lol. So its some sort of ridiculous macOS bug that breaks the os when devs are allowed... it must have never happened on the old machine because I didnt migrate anything onto that one and just started from scratch. Also safe mode would not work. only recovery mode. Just letting you know in case anyone else comes crying to ya :-) thanks for your work btw I like having fuse a lot.
